README.md

```diff
@@ -43,6 +43,11 @@ or
 ```vim
 :Telescope luasnip
 ```
+for windows system
```

Comment:
Hi; do we still need this change to the README, if we include your change to `luasnip.lua`?

Reply:
In fact, it can not be reflected in the readme, but I want to use the user until there is an optional configuration, and this configuration can improve the switching speed of the preview buffer when the highlighting is disabled. Maybe we can use another form.